### PR TITLE
added Simplified Chinese localization

### PR DESCRIPTION
**Pull request by L1cardo**
I have localized all localizable strings, but there are also some that are not.

截屏2019-07-05下午1 59 11

截屏2019-07-05下午1 59 25

截屏2019-07-05下午2 00 17

### CODE REVIEW

File: Pock/Widgets/NowPlaying/NowPlayingItemView.swift [hunk 1]
```diff
@@ -36,7 +36,7 @@ class NowPlayingItemView: PKDetailView {
         switch (appBundleIdentifier) {
         case "com.apple.WebKit.WebContent":
             appBundleIdentifier = "com.apple.Safari"
-        case "com.spotify.client", "com.apple.iTunes", "com.apple.Safari":
```
Comment (L1cardo) [inline Pock/Widgets/NowPlaying/NowPlayingItemView.swift]: added [Netease music](https://music.163.com/) support , which is the most common music app in China

**Comment by pigigaldi** [comment]
You need to localize the project first and then use [NSLocalizedString](https://developer.apple.com/documentation/foundation/nslocalizedstring).
You can find a helpful article [here](https://medium.com/@marcosantadev/app-localization-tips-with-swift-4e9b2d9672c9), written by @MarcoSantarossa, that can give you some excellent tips on how to handle localization with Swift.

**Comment by L1cardo** [comment]
> You need to localize the project first and then use [NSLocalizedString](https://developer.apple.com/documentation/foundation/nslocalizedstring).
> You can find a helpful article [here](https://medium.com/@marcosantadev/app-localization-tips-with-swift-4e9b2d9672c9), written by @MarcoSantarossa, that can give you some excellent tips on how to handle localization with Swift.

I made changes based on your suggestions. Now it works perfect !

截屏2019-07-06下午9 05 17

截屏2019-07-06下午9 05 28

截屏2019-07-06下午9 05 56

**Comment by L1cardo** [comment]
Another issue is that I don't know what these strings below stands for, and I haven't met them yet , so it is not localized.

截屏2019-07-06下午9 34 20

**Comment by pigigaldi** [comment]
Hey @L1cardo, I want to merge your PR into Pock because I know how better it could be if more localizations will be added. 
Sadly, I have to ask you to refactor your code in a way that, if Apple changed `NSLocalizedString` structure, we'd not need to mess around the project to fix all cases.

What do you think about something like this?
```
extension String {
    var localized: String {
        return NSLocalizedString("Dock Widget", comment: "Dock Widget")
    }
}
```

**Comment by L1cardo** [comment]
I want to do better than now, but actually I am a student of a medical school, sadly that I know a little about coding. Most of that is beyond my understanding. what I have completed , I compeleted them very hard. So can you do the coding and just leave the localization to me? You can modify the structure and make localization easier , and then I will add my localization . It is ok with me to do it over , but coding is really hard for me.------------------ 原始邮件 ------------------
发件人: "Pierluigi Galdi"<notifications@github.com>
发送时间: 2019年7月7日(星期天) 凌晨2:15
收件人: "pigigaldi/Pock"<Pock@noreply.github.com>;
抄送: "Licardo"<1014660822@qq.com>;"Mention"<mention@noreply.github.com>;
主题: Re: [pigigaldi/Pock] added Simplified Chinese localization (#134)



Hey @L1cardo, I want to merge your PR into Pock because I know how better it could be if more localizations will be added.
 Sadly, I have to ask you to refactor your code in a way that, if Apple changed NSLocalizedString structure, we'd not need to mess around the project to fix all cases.
 
What do you think about something like this?
 extension String {     var localized: String {         return NSLocalizedString("Dock Widget", comment: "Dock Widget")     } }  
&mdash;

**Comment by L1cardo** [comment]
How about now ?  This is the best I can do .

**Pull request merged**